src/System.Private.CoreLib/shared/System/Array.cs

```diff
@@ -936,9 +936,8 @@ public static int IndexOf(Array array, object? value, int startIndex, int count)
                 return retVal;
 #endif
 
-            object[]? objArray = array as object[];
             int endIndex = startIndex + count;
-            if (objArray != null)
+            if (array is object[]objArray)
```

Comment:
space needed on both changes in this file.

Reply:
Fixed